web/client/components/widgets/widget/ControlledPopover.jsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, {useState, useRef} from 'react';
+import PropTypes from 'prop-types';
+import {
+    Popover,
+    Glyphicon,
+    Button
+} from 'react-bootstrap';
+
+import Overlay from '../../misc/Overlay';
+/**
+ * ControlledPopover. A component that renders a icon with a Popover
+ * it uses:
+ * - mouseEnter on glyph to open
+ * - click to close
+ * it supports fixed text with clickable link
+ * @prop {string} title the title of popover
+ * @prop {string|function} text the text of popover (can be a component, like I18N.HTML)
+ * @prop {string} glyph glyph id for the icon
+ * @prop {number} left left prop of popover
+ * @prop {number} right right prop of popover
+ * @prop {string} placement position of popover
+ */
+const ControlledPopover = ({
+    text,
+    placement,
+    bsStyle,
+    left,
+    top,
+    title,
+    glyph,
+    id
+}) => {
+    const trigger = useRef();
+    const [show, setShow] = useState(false);
+    return (
+        <span className="mapstore-info-popover">
```

Comment:
```suggestion
       <span className="mapstore-info-popover" onMouseLeave={() => setShow(false)}>
```
Adding this you can remove the close button

Reply:
yes great, seems a good hint